app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
When importing multiple classes or modules, it is recommended to use parentheses to improve readability. According to the checklist, you should use parentheses for such imports.